src/JitInterface/src/CorInfoHelpFunc.cs

```diff
@@ -305,6 +305,8 @@ public enum CorInfoHelpFunc
         CORINFO_HELP_JIT_REVERSE_PINVOKE_ENTER, // Transition to cooperative mode in reverse P/Invoke prolog, frame is the first argument
         CORINFO_HELP_JIT_REVERSE_PINVOKE_EXIT,  // Transition to preemptive mode in reverse P/Invoke epilog, frame is the first argument
 
+        CORINFO_HELP_READYTORUN_GVMLOOKUP_FOR_SLOT,
```

Comment:
Probably makes more sense removing the "READYTORUN" portion from the name, and having the jit just call the getHelperFtn API to get the helper instead of calling the getReadyToRunHelper API...